env-config/live.env

```diff
@@ -1,4 +1,4 @@
-SIMORGH_BASE_URL=https://www.bbc.com
+SIMORGH_BASE_URL=https://www.test.bbc.com
```

Comment:
What's the reasoning for changing this to point to Test?

Reply:
Ask spoke separately, this is needed because the bake scripts no longer point the LIVE config to the TEST env (pretty hacky logic in the first place). I've opened the issue https://github.com/bbc/simorgh/issues/1254 to ensure this is changed back once the LIVE routing and data source are available.